app/src/main/res/values/styles.xml

```diff
@@ -7,6 +7,12 @@
         <item name="colorAccent">@color/colorAccent</item>
     </style>
 
+    <style name="AppTheme.NoActionBar">
```

Comment:
be careful here, because there already exists a native NoActionBar theme

Reply:
Yes! Refactored!